test/test-applications/security/Samples.AspNetCore5/Properties/launchSettings.json

```diff
@@ -40,7 +40,7 @@
         "CORECLR_PROFILER_PATH": "$(ProjectDir)$(OutputPath)profiler-lib\\Datadog.Trace.ClrProfiler.Native.dll",
         "DD_VERSION": "1.0.0",
         "DD_TRACE_HEADER_TAGS": "sample.correlation.identifier, Server",
-        "DD_TRACE_DEBUG": "0"
+        "DD_TRACE_DEBUG": "1"
```

Comment:
Was this intentional or were you just testing locally?

Reply:
Good catch, local debug, fixed in 702cc956